downstream/assemblies/platform/assembly-network-ports-protocols.adoc

```diff
@@ -103,6 +103,30 @@ ALLOW connection from controller(s) to Receptor port |
 * Hybrid nodes act as a combination of control and execution nodes, and therefore Hybrid nodes share the connections of both. 
 
 * If `receptor_listener_port` is defined, the machine also requires an available open port on which to establish inbound TCP connections, for example, 27199.
+
+* It might be the case that some servers do not listen on receptor port (the default is 27199)
++
+Suppose you have a  Control plane with nodes A, B, C, D
++
+The RPM installer creates a strongly connected peering between the control plane nodes with a least privileged approach and opens the tcp listener only on those nodes where it is required. All the receptor connections are bidirectional, so once the connection is created, the receptor can communicate in both directions. 
++
+The following is an example peering set up for three controller nodes 
++
+Controller node A --> Controller node B
++
+Controller node A --> Controller node C
++
+Controller node B --> Controller node C
++
+You can force the listener by setting
++
+`receptor_listener=True`
++
+However, a connection Controller B --> A is likely to be rejected as that connection already exists.
```

Comment:
I don’t love saying is likely here. I’m unsure of the installer behavior but this is something we can test manually.

im fine if we leave it like this and circle back at a later time to address this issue